parser_test.go

```diff
@@ -706,6 +706,26 @@ func TestParser_Parse(t *testing.T) {
 			},
 			err: nil,
 		},
+		{
+			name: "parse rule with jump target SNAT",
+			s:    "-A foo -o eth0 -4 -j SNAT --to-source 192.168.1.1",
```

Comment:
can you add a test case where `--to-source` is not the last flag?

Reply:
It's done 😉